lib/pam/pam_test.go

```diff
@@ -91,19 +92,11 @@ func (s *Suite) TestEcho(c *check.C) {
 // read in the first argument and set it as a PAM environment variable. This
 // test then validates it matches what was set in the policy file.
 func (s *Suite) TestEnvironment(c *check.C) {
-	// Skip this test if the binary was not built with PAM support.
-	if !BuildHasPAM() || !SystemHasPAM() {
-		c.Skip("Skipping test: PAM support not enabled.")
-	}
-
-	local, err := user.Current()
-	c.Assert(err, check.IsNil)
-
```

Comment:
Are we now assuming that every single binary is built with PAM support, so tests will fail if it's not present? Or is this controlled by the `pam` tag?

Reply:
This check is performed in `SetUpSuite`.
If the skip happens there, no tests will run.
There's no need to do the same check in every test now.